framework/src/consumption/lib/opensearch/opensearch.ts

```diff
@@ -61,14 +61,15 @@ export class OpenSearchCluster extends TrackedConstruct {
   public readonly vpc:IVpc | undefined;
 
   /**
-   * CDK Custom resource provider for calling OpenSearch APIs
```

Comment:
Can you expose the resource created by the DSFProvider from the opensearch api construct.